src/xdocs/report_issue.xml

```diff
@@ -193,6 +202,12 @@ Describe how the additional functionality will solve the problem:
 
 Additional context, screenshots or examples:
       </source>
+      <p>
+        More Examples:
+        <a href="https://github.com/checkstyle/checkstyle/issues/8764#issue-686630541">
```

Comment:
this is not good example, we need issue for new property into existing Check.

Reply:
@romani Is it a good issue? It tries to implement a new feature inside already existed check(lambda, Indentation)
https://github.com/checkstyle/checkstyle/issues/8341

Reply:
lets use https://github.com/checkstyle/checkstyle/issues/3201
and https://github.com/checkstyle/checkstyle/issues/6582

I found it by scrolling https://checkstyle.org/releasenotes.html for "new:"

Reply:
Nice hack. I have added these example, all two of them are good!